src/components/DoodleVideo.vue

```diff
@@ -100,13 +152,15 @@ export default {
     },
     async fetchStrokes () {
       return new Promise(async resolve => {
-        const strokesRef = this.blackboardRef.collection("strokes").orderBy("strokeNumber", "asc");
-        this.allStrokes = await helpers.getCollectionFromDB(strokesRef);
-        this.$nextTick(() => {
-          this.hasFetchedStrokes = true;
-          this.$_drawMixin_rescaleCanvas(true);
-          this.$emit("strokes-ready")
-        });
+        if(!this.hasFetchedStrokes){
```

Comment:
I think it can be better to assume the pre-condition that strokes haven't been fetched yet, and instead let it be the parent's responsibility to do the check